dcm4ceph-core/src/main/java/org/open_ortho/dcm4ceph/core/Cephalogram.java

```diff
@@ -285,6 +291,7 @@ public void setFromProperties(Properties cephprops) {
                                     + cephprops.getProperty("studyTime")));
         } catch (ParseException e) {
             e.printStackTrace();
+            Log.warn("using current datetime as studyDateTime");
```

Comment:
Make log message consistent with previous one: "Could not parse Study Date Time correctly. Using current date time."

Reply:
Done